src/main/kotlin/fi/metatavu/lipsanen/users/UsersApiImpl.kt

```diff
@@ -134,4 +175,32 @@ class UsersApiImpl: UsersApi, AbstractApi() {
         userController.deleteUser(user)
         createNoContent()
     }
+
+    /**
+     * Checks if project owner can assign user to projects.
+     * It can assign users to the projects that he/she is project owner of.
+     * He cannot un-assing user from the projects that he/she is not project owner of.
+     *
+     * @param updatingUser user who is updating the user
+     * @param updatableUser user who is being updated
```

Comment:
```suggestion
     * @param updatedUser user who is being updated
```
Maybe updatedUser is more suitable here?